drivers/platform/stm32/stm32_tdm.c

```diff
@@ -219,3 +228,46 @@ int32_t stm32_tdm_read(struct no_os_tdm_desc *desc,
 
 	return ret;
 }
+
+/**
+ * @brief Stop SAI DMA transfer
+ * @param desc - The TDM descriptor.
+ * @return 0 in case of success, negative error code otherwise.
+ */
+int32_t stm32_stop_tdm_dma_transfer(struct no_os_tdm_desc *desc)
+{
+	int32_t	ret;
+	struct stm32_tdm_desc *tdesc;
+
+	if (!desc)
+		return -EINVAL;
+
+	tdesc = desc->extra;
+
+	ret = HAL_SAI_DMAStop(&tdesc->hsai);
+	if (ret)
+		return ret;
+
+
+	return 0;
+}
+
+/**
+ * @brief Pause SAI DMA transfer
+ * @param desc - The TDM descriptor.
+ * @return 0 in case of success, negative error code otherwise.
+ */
+int32_t stm32_pause_tdm_dma_transfer(struct no_os_tdm_desc *desc)
+{
+	return 0;
```

Comment:
resume/pause need to be implemented just like the stop

Reply:
Resume and Pause are also implemented now.